### PR TITLE
Fix alignment in edit profile page

### PR DESCRIPTION
**Pull request by ryuk156**
Fixes  https://github.com/CircuitVerse/CircuitVerse/issues/1143

#### Describe the changes you have made in this PR -
Added some media queires  and made some other changes set width of select to 100% which was not looking good due to max -width and 

### Screenshots of the changes (If any) -
before:
image

After:
image

image

### CODE REVIEW

**Comment by yash15601** [comment]
@nitin10s  @aayush-05  why this trailing error can suggest me something will squash the commit later

File: app/assets/stylesheets/profile.scss [hunk 1]
```diff
@@ -1,3 +1,41 @@
...
+
+@media (max-width:576px) { 
+  .container-padding {
+    padding-left: 0;
+  }
+  .profile-text-feild {
+    height: 40px;
+    width: 275px;
+  }
+}
```
[the remark this answers is not in the document]
Comment (yash15601) [reply app/assets/stylesheets/profile.scss]: Done

File: app/assets/stylesheets/profile.scss [hunk 2]
```diff
@@ -1,3 +1,43 @@
...
+    width: 275px;
+    height: 40px;
+  }
+}
+
+
+@media (max-width: 768px) { 
+  .container-padding {
+    padding-left: 0;
+  }
```
[the remark this answers is not in the document]
Comment (yash15601) [reply app/assets/stylesheets/profile.scss]: Done

File: app/assets/stylesheets/profile.scss [hunk 3]
```diff
@@ -1,3 +1,43 @@
...
+
+.profile-text-feild {
+  width: 100%;
+  height: 50px;
+}
+
+@media (max-width: 568px) {
+  .container-padding {
+    padding-left: 0;
+  }
```
[the remark this answers is not in the document]
Comment (yash15601) [reply app/assets/stylesheets/profile.scss]: Done

File: app/assets/stylesheets/profile.scss [hunk 4]
```diff
@@ -1,3 +1,43 @@
...
+  .profile_text_feild {
+    width: 250x;
+    height: 40px;
+  }
+}
+
+@media (max-width: 576px) { 
+  .container-padding {
+    padding-left: 0;
+  }
```
[the remark this answers is not in the document]
Comment (yash15601) [reply app/assets/stylesheets/profile.scss]: Done

**Comment by yash15601** [comment]
@aayush-05  @nitin10s  review please

**Comment by yash15601** [comment]
@tachyons  @nitin10s  @aayush-05  Review please

**Comment by yash15601** [comment]
@nitin10s  review please

**Comment by aayush-05** [comment]
Add screenshots for pc as well. Also see if there is heading tag getting used for the field labels, use paragraph and strong ones instead (text on this page is extra large as compared to other pages)

**Comment by yash15601** [comment]
@aayush-05  The  heading tags look great i tryed with paragraph and strong tags it looks bit odd and the Sreenshot is already their for pc

**Comment by yash15601** [comment]
@aayush-05 i tryed tag using h5 does it look good now before i was using h4
image

**Comment by aayush-05** [comment]
> @aayush-05 i tryed tag using h5 does it look good now before i was using h4
Use paragraph with strong tag for now, to maintain uniformity.

**Comment by yash15601** [comment]
image
@aayush-05  done @tachyons  @nitin10s  review  please

**Comment by yash15601** [comment]
@tachyons  Review please

File: app/views/users/logix/edit.html.erb [hunk 5]
```diff
@@ -32,12 +32,12 @@
 
         <div class="field form-group">
          <%= f.label :country, { class: 'h4 font-weight-bold'} %><span> [Optional]</span><br/>
-          <%= f.country_select :country, { include_blank: 'Prefer Not To Enter', priority_countries: get_priority_countries() }, { class: "form-control rounded-0 border-success", style:"width:100%; height:50px" } %>
+          <%= f.country_select :country, { include_blank: 'Prefer Not To Enter', priority_countries: get_priority_countries() }, { class: "form-control rounded-0 border-success profile-text-feild" } %>
         </div>
 
         <div class="field form-group">
           <%= f.label :educational_institute , { class: 'h4 font-weight-bold'} %><span> [Optional]</span><br/>
-          <%= f.text_field :educational_institute,{ class: "typeahead form form-group form-control rounded-0 border-success", style:"width:100%; height:50px",autocomplete:"off" } %>
```
[the remark this answers is not in the document]
Comment (yash15601) [reply app/views/users/logix/edit.html.erb]: should  i remove autocomplete

**Comment by yash15601** [comment]
@nitin10s your views please

**Comment by yash15601** [comment]
@tachyons  i checked it is wroking properly  Review please

**Comment by tachyons** [comment]
@yash15601 screenshot for typeahead feature please

**Comment by yash15601** [comment]
image

@tachyons

**Comment by yash15601** [comment]
Screenshot (82)
@tachyons  review

**Comment by yash15601** [comment]
@tachyons  Review This please

**Comment by yash15601** [comment]
@nitin10s  done

**Comment by yash15601** [comment]
@nitin10s  @tachyons  @satu0king  review please

**Comment by aayush-05** [comment]
Can you try to fix hound issues, rather than just marking them resolved @yash15601

**Comment by yash15601** [comment]
@aayush-05  will fix this soon

**Comment by yash15601** [comment]
@nitin10s  review please

**Comment by tachyons** [comment]
@aayush Check if this PR is still relevant

**Comment by aayush-05** [comment]
This can be closed @tachyons; everything fixed here >> #1501

**Comment by tachyons** [comment]
Thanks  ryuk156 for your efforts

**Pull request closed without merging**